tests/Dockerfile

```diff
@@ -24,7 +24,8 @@ RUN mkdir benches
 RUN touch benches/my_benchmark.rs
 
 # This is a dummy build to get dependencies cached
-RUN cargo build --release
+RUN rustup install stable
+RUN rustup run stable cargo build --release
```

Comment:
I am honestly not sure about the significance of this? Could you expand a little why you made this change? Wouldn't the version of rust specified in the docker image default to stable?